build.gradle

```diff
@@ -78,10 +78,34 @@ subprojects {
           artifact sourcesJar
 
           pom {
+            name = 'Google Cloud Common Protos for Java'
```

Comment:
Just curious,iIs this copied (then modified) from gax-java?

Reply:
hahah sure was!